src/shared/components/NotFound.tsx

```diff
@@ -4,7 +4,7 @@ const NotFound: FC = () => (
   <div className="container-fluid" data-testid="not-found">
     <div className="panel">
       <div className="panel-heading text-center">
-        <h1 className="deluxe">404</h1>
```

Comment:
dont need this change. if the 404 is the only thing rendered, then we want to keep the header, as it is now the page

Reply:
fixed